dataland-frontend/.env.e2e

```diff
@@ -1 +1,3 @@
-NODE_ENV=e2e
\ No newline at end of file
+NODE_ENV=e2e
+VUE_APP_API_URL="http://localhost:8080"
```

Comment:
Bitte umbennennen, z.B. in VUE_APP_API_BASE_URL - es wir ja noch "/api" angehängt"

Reply:
Umbenannt

Reply:
Die Variable gibts (mindestens) noch einmal in der alten Benennung in `DataStore.ts`

Reply:
Fixed, Danke!

Reply:
ok